### PR TITLE
[Diagnostics] Adding history.md files for both AspNet and AspNetCore

### PR DESCRIPTION
**Pull request by amanda-tarafa**

### CODE REVIEW

File: apis/Google.Cloud.Diagnostics.AspNet/docs/history.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Version history
+
+# 3.0.0-beta07, 2018-11-12
+
+New features since 2.0.0:
+
+- When logging exceptions, we have stopped logging HTTP Status Code information since we were storing inconsistent values.
+- Adding GoogleWebApiExceptionLogger which supports proper exception logging, including HTTP context information for ASP.NET Web Api applications.
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNet/docs/history.md]: Api => API?

File: apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+New features since 2.0.0:
+
+- When logging exceptions, we have stopped logging HTTP Status Code information since we were storing inconsistent values.
+- Supporting the .NET Standard 2.0 `System.Net.Http.IHttpClientFactory` for creating `System.Net.Http.HttpClient` instances when tracing outgoing requests.
+- Adds format parameters from all parent log scopes in a Log Entry.
+- Out of the box Log Entries Label Providers.
+- Hook to allow for custom labels to be added to Log Entries.
+- Convenience extension method for registering Google Diagnostics in `Microsoft.AspNetCore.Hosting.IWebHostBuilder`.
+- By default all Log Entries are stored in one single Log on Stackdriver.
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md]: Log Entries => log entries and Log => log?

File: apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Version history
+
+# 3.0.0-beta07, 2018-11-12
+
+New features since 2.0.0:
+
+- When logging exceptions, we have stopped logging HTTP Status Code information since we were storing inconsistent values.
+- Supporting the .NET Standard 2.0 `System.Net.Http.IHttpClientFactory` for creating `System.Net.Http.HttpClient` instances when tracing outgoing requests.
+- Adds format parameters from all parent log scopes in a Log Entry.
+- Out of the box Log Entries Label Providers.
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md]: Log Entries Label Providers => log entries label providers?

File: apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Version history
+
+# 3.0.0-beta07, 2018-11-12
+
+New features since 2.0.0:
+
+- When logging exceptions, we have stopped logging HTTP Status Code information since we were storing inconsistent values.
+- Supporting the .NET Standard 2.0 `System.Net.Http.IHttpClientFactory` for creating `System.Net.Http.HttpClient` instances when tracing outgoing requests.
+- Adds format parameters from all parent log scopes in a Log Entry.
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md]: Log Entry => log entry?

File: apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- Out of the box Log Entries Label Providers.
+- Hook to allow for custom labels to be added to Log Entries.
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md]: Log Entries => log entries

File: apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- Convenience extension method for registering Google Diagnostics in `Microsoft.AspNetCore.Hosting.IWebHostBuilder`.
+- By default all Log Entries are stored in one single Log on Stackdriver.
+- Add Trace IDs to Log Entries if they are available.
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md]: Trace => trace, and Log Entries => log entries

File: apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+# 2.0.0, 2017-11-27
+
+New features since 1.0.0:
+
+- Make Google Project ID optional when creating a Google Logger.
+- Support for recommended way of adding Logging Providers in ASP.NET Core 2.0
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/docs/history.md]: Logging Providers => logging providers?

**Comment by amanda-tarafa** [comment]
Thanks! All changes made. Pushing a commit and merging.

**Pull request merged**